src/Messenger.Client/Objects/MessengerSenderAction.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+
+namespace Messenger.Client.Objects
+{
+    public enum MessengerSenderAction
+    {
+        MarkSeen,
+        TypingOn,
+        TypingOff,
+    }
+
+
+    internal static class MessengerSenderActionExtensions
```

Comment:
Let's stick to one file one thing rule and place it in a separate file

Reply:
Sounds good 👍🏻